xep-0353.xml

```diff
@@ -237,48 +431,123 @@
 ]]></example>
   </section2>
   <section2 topic='Finishing a Started Session' anchor='finish'>
-    <p>This protocol in conjunction with &xep0280; and &xep0313; allows all devices of both involved parties to get synchronized about session start, rejection etc. To synchronize the ending of the session, both parties MUST send a message stanza containing a &lt;finish/&gt; element specifying the same session ID as in <link url='#accept'>Accept</link> to the bare jid of the other party.</p>
-    <p>Letting both involved parties send the &lt;finish/&gt; element makes sure we have the correct state in MAM archives etc. even if one client suddenly looses connectivity/power. It even makes possible for a client to determine if the call is still deemed "running" by the other party if it manages to recover from connectivity loss &mdash; before the other party runs into a timeout and sends a &lt;finish/&gt; &mdash; to recover the session or formally terminate the call (by ending the Jingle session and sending a &lt;finish/&gt; message itself). See <link url="#tie-break-2">Tie Breaking</link> for more infos on this and similar scenarios.</p>
-    <p>The &lt;finish/&gt; element MUST contain a &lt;reason/&gt; element as defined in &xep0166; section 7.4. This SHOULD use a condition of &lt;success/&gt;, but implementations MAY use other conditions if deemed more appropriate (see <link url="#security">Security Considerations</link> below for details and rationale).</p>
-    <example caption="Initiator Sends Finish Message"><![CDATA[
+    <p>
+      This protocol in conjunction with &xep0280; and &xep0313; or appropriate
+      groupchat protocols (like &xep0045;) allows all devices of all involved
```

Comment:
I don't see how this solves a problem in MUC where the full JIDs are known.

Reply:
I don't understand what you mean here. Synchronizing the call state via messages works independently of full JIDs being used in MUCs or not. MUCs do however help with synchronization by providing message history (although nowadays we typically use MAM there as well) and forwarding the messages to all other participating resources in the room (which include your own other devices).